CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ option (WITH_HEADERS                        "Install header files (to build samp
 option (WITH_SAMPLES                        "Install sample files (requires headers)"               ON)
 option (WITH_TESTS                          "Install unit tests and benchmarks"                     ON)
 option (WITH_PYTHON                         "Build Python bindings"                                 ON)
```

Comment:
I suggest updating the help message to clarify that this will build Python 2 bindings only.